js/config.js

```diff
@@ -4,6 +4,7 @@ let Config = {
 
     ApiServerHost: "https://esapi.isthereanydeal.com",
     CdnHost: "https://es.isthereanydeal.com",
-    PublicHost: "//es.isthereanydeal.com"
+    PublicHost: "//es.isthereanydeal.com",
+    ITAD_ApiServerHost: "https://api.isthereanydeal.com"
```

Comment:
Camelcase and snakecase, interesting

Reply:
Actually, I use this combination when I have a word that ends with a capital letter